docs/examples/Creating_Actionable_Sensor.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python
```

Comment:
Oli made some changes to the example since this review - I think the content is now good to be merged with this PR but I agree it could still be improved with additional diagrams etc. I have created an issue for this #633 so I remember to revisit it.